fairlearn/metrics/_multi_curve_plot.py

```diff
@@ -0,0 +1,126 @@
+from ..utils._input_validation import _validate_and_reformat_labels, _validate_and_reformat_labels_and_sf, check_consistent_length
+from ..postprocessing._plotting import _MATPLOTLIB_IMPORT_ERROR_MESSAGE
+from ._make_derived_metric import _DerivedMetric
+from typing import Callable, List, Union
+
+def plot_model_comparison(
+    *,
+    x_axis_metric: Callable[..., Union[float, int]],
+    y_axis_metric: Callable[..., Union[float, int]],
+    y_true,
+    y_preds,
+    sensitive_features,
+    show_plot: bool,
+    ax=None,
+    **kwargs
+    ):
+    """Plot a model comparison
```

Comment:
>     * [ ]  API entry from numpydoc???

The docstring you're writing is doing that already. Or did you mean something else?

Reply:
I think this may be the same question?

>     * How do I add this numpydoc to the API?

The API reference is generated from the numpydoc strings in each code file so this is automatic. You can run the doc build locally to see what it'll look like: https://fairlearn.org/main/contributor_guide/contributing_documentation.html

Reply:
Ah, I need glasses. I figured this, but could not find it in the docs. Now that I search for it I can find it :D